app/policies/adoptable_pet_policy.rb

```diff
@@ -6,7 +6,7 @@ class AdoptablePetPolicy < ApplicationPolicy
   end
 
   def show?
-    allow! if permission?(:manage_pets)
+    allow! if allowed_to?(:manage?, record, namespace: Organizations)
```

Comment:
This utilizes Action Policy to call a policy rule from within another policy. The permission check was fine as well, but this means that other logic besides the permission check are also carried over.